lib/screens/formulario_transferencia.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:bytebank/models/contato.dart';
+import 'package:bytebank/models/transferencia.dart';
+import 'package:bytebank/webclient/transferencia_webclient.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+const _tituloAppBar = 'Nova transferência';
+
+const _rotuloCampoValor = 'Valor';
+const _tituloBotaoCriar = 'Transferir';
+
+class FormularioTransferencia extends StatefulWidget {
+  final Contato _contato;
+
+  @override
+  State<StatefulWidget> createState() {
+    return _FormularioTransferenciaState();
+  }
+
+  FormularioTransferencia(this._contato);
+}
+
+class _FormularioTransferenciaState extends State<FormularioTransferencia> {
+  final TextEditingController _controlador = TextEditingController();
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(_tituloAppBar),
+      ),
+      body: Column(
+        children: <Widget>[
+          Padding(
+            padding: const EdgeInsets.all(16.0),
```

Comment:
You are repeating this horizontal padding everywhere
Why not just wrap this whole Column inside a Padding, then add the vertical ones to the children individually?

Also, if the device is small, this might not fit. You could try some sort of scrolling widgets here to guarantee that behavior (and they have a padding property as well)

Reply:
I adjusted the padding of each child using symmetric modifing the `vertical` property and added the padding whole Column Widget for horizontal edges. For the scrolling I considired the `SingleChildScrollView`, ok?